src/util/config/global-path.ts

```diff
@@ -17,7 +38,16 @@ const getNowDir = () => {
   const customPath = args['global-config'];
 
   if (!customPath) {
-    return path.join(homedir(), '.now');
+    const nowDir = path.join(
+      process.env.XDG_DATA_HOME || path.join(homedir(), '.local', 'share'),
```

Comment:
How does this standard affect Windows? Or if it doesn't apply, is there a better place for us to store these files on Windows? Perhaps `$HOME/AppData/co.zeit.now/Credentials`?

Reply:
You might look at [xdg-app-paths](https://www.npmjs.com/package/xdg-app-paths) / [xdg-portable](https://www.npmjs.com/package/xdg-portable) (both written by me) which allows use of XDG on any platform with platform-compatible defaults. Specifically, [xdg-portable](https://www.npmjs.com/package/xdg-portable) has and discusses reasonable platform-dependent defaults.

Reply:
I'm using `xdg-portable` now and if nothing can be found the old location is used (which should not happen on normal systems I think)